expenses-manager-backend/src/main/java/com/endava/expensesmanager/service/UsersService.java

```diff
@@ -0,0 +1,46 @@
+package com.endava.expensesmanager.service;
+
+import com.endava.expensesmanager.entity.Users;
+import com.endava.expensesmanager.repository.UsersRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class UsersService {
```

Comment:
Iti recomanda ca pentru fiecare serviciu sa creezi o interfata si in locurile in care avem nevoie de serviciu sa depindem de interfata, nu de implementare. Acest lucru ne va ajuta pe viitor in caz ca dorim sa scriem teste, aici poti gasi un exemplu https://davidgiard.com/java-services-and-interfaces-in-a-spring-boot-application